src/searchModal/advancedSearch.js

```diff
@@ -64,13 +65,19 @@ export default function advancedSearchFactory(config) {
          * @returns {Promise} - Request promise
          */
         updateCriteria: function (route) {
+            const $criteriaIcon = $('.add-criteria-container a span').eq(0);
```

Comment:
The wrapper of an icon should be button, not an empty link.

Reply:
https://oat-sa.atlassian.net/browse/ADF-293 To add not only this, if you found more accessibility issues do all at same time